nsqd/protocol_v2.go

```diff
@@ -92,6 +93,7 @@ func (p *protocolV2) IOLoop(conn net.Conn) error {
 
 			sendErr := p.Send(client, frameTypeError, []byte(err.Error()))
 			if sendErr != nil {
+				p.ctx.nsqd.logf("ERROR: [%s] - %s%s", client, sendErr, ctx)
```

Comment:
should we do something like we do on https://github.com/nsqio/nsq/pull/670/files#diff-12a30208ec1e011f7cd53fe10c77575cR110 instead?